argus/src/intTest/README.md

```diff
@@ -10,3 +10,84 @@ for a CADC staff member to run. Required certificates:
 The _auth_ identity has to be the (CADC) staff member's personal certificate because the test tries to output
 to `vos://cadc.nrc.ca~vault/{user.name}/test/some-file-name`. The _noauth_ identity should be cadcregtest1.
 
+An example of a minimum configuration required for the `argus` service to run on a local machine consists of the
+following containers:
+- `haproxy` (image: `cadc-haproxy-dev`) - for https termination
+- `postgress` (`cadc-postgresql-dev`) - database container
+- `reg` (`reg`) - local registry entries
+- `icewind` (`icewind`) - to create the `caom2` schema and tables
```

Comment:
technically, one can use `icewind` to create the caom2 tables and add some content **OR** deploy `torkeep` to just create the tables

I do not recall if the tests will all pass when the tables are empty, but they should. The tests would not hit all possible code paths, but that's going to be true with real content and would not be completely thorough unless we specify some specific test content (a handful of observations with all possible fields filled in).